### ISSUE TITLE
[Kubernetes]: [Metrics] Kubernetes Overview dashboard aggregation leads to max_buckets exceed

### ISSUE THREAD

**Issue by Alphayeeeet** [elastic/integrations#11464]
### Integration Name

Kubernetes [kubernetes]

### Integration Version

1.68.1

### Agent Version

8.15.2

### Agent Output Type

elasticsearch

### Elasticsearch Version

8.15.2

### OS Version and Architecture

RHEL 8.9

### Error Message

Visualization: error while executing search
Control element: Internal Server Error (at least sometimes, as I couldn't reproduce for screenshot).

The Elasticsearch log prints out this message:
![Image](https://github.com/user-attachments/assets/f092df87-879d-47fe-b8bb-1c355a32f781)

### What did you do?

After some of the previous integration updates (presumably https://github.com/elastic/integrations/pull/10593), the newly added visualizations exceed the bucket limit, and break also the filter control element (sometimes, as I couldn't reproduce when creating the screenshot.). The control element for "Cluster Name" then said: "Internal Server Error.

### What did you see?

A uncompletely loaded dashboard like this:
![Image](https://github.com/user-attachments/assets/49ac3ea0-5c7d-433a-ab6c-710d6c5f2251)

### What did you expect to see?

A fully loaded dashboard like this:

![Image](https://github.com/user-attachments/assets/950981f5-59a6-4d33-8c90-9852ac7ca80b)

### Anything else?

Important side node: The time range in all the above screenshots is past 15 min. Longer timeranges not even considered.

Also: Increasing the max_bucket setting helped. But only after increasing it excessively to > 100k and < 1 mio (I didn't try any in between).

**Comment by Alphayeeeet** [comment]
Suggesting also somehow, to set a different data view in the built-in dashboards, as metrics-* can be quite large in a fully built production environment. This could be a performance imporvement for future, so please consider creating a separate issue in the affected projects for this.

**Comment by Alphayeeeet** [comment]
+1
